<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
   </property>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>521</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/park.svg</pixmap>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background: rgb(235,  235, 235);}</string>
    </property>
    <property name="text">
     <string>9991</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="hostLineEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background: rgb(235,  235, 235);}</string>
    </property>
    <property name="text">
     <string>emb2.extcam.xyz</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>161</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: white;
	borderpstyle: outset;
}</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="connectToServer">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>19</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color : red; }</string>
     </property>
     <property name="text">
      <string>Not connected</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusStringLabel">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>19</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color : white; }</string>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="hostLabel">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>50</y>
       <width>43</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color : white; }</string>
     </property>
     <property name="text">
      <string>Host : </string>
     </property>
    </widget>
    <widget class="QLabel" name="portLabel">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>89</y>
       <width>44</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color : white; }</string>
     </property>
     <property name="text">
      <string>Port :  </string>
     </property>
    </widget>
   </widget>
   <zorder>image</zorder>
   <zorder>connectToServer</zorder>
   <zorder>hostLineEdit</zorder>
   <zorder>portLineEdit</zorder>
   <zorder>connectButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
